readme.md

```diff
@@ -111,10 +127,12 @@ try {
 
 ## API
 
-### execa(file, [arguments], [options])
+### execa(file | command, [arguments], [options])
```

Comment:
I think it would be clearer to define this as overloads:

```js
### execa(file, [arguments], [options])
### execa(command, [options])
```

Reply:
Maybe we should throw a useful error if `arguments` is given and `command` has spaces?